### ISSUE TITLE
Load types dynamically.

### ISSUE THREAD

**Issue by aronhelser** [Kitware/simput#49]
The new Vue.js UI is loading types statically using webpack. @jourdain  says:

> the thing that is not worked out yet is how to register your type like we do it [here](https://github.com/Kitware/simput/blob/master/static/index.html#L34-L37).
Knowing that those types could be hosted anywhere... 

The simput cli no longer helps with this.

**Comment by jourdain** [comment]
I'm not sure to understand what you mean by `statically using webpack`. Since it is dynamic and left to the Simput integrator to run the registration on their own. In fact maybe what could be missing in the current app, is a form to register types that are available over http.

**Comment by aronhelser** [comment]
Yes, I don't know how to pose the problem. Feel free to edit or reword!

**Comment by jourdain** [comment]
That's a none issue as it should be part of the simput integration in some way. Although it could be nice to have dynamic registration via the Simput CLI for users that want to use the application locally with their private type.
